aloscene/scene_flow.py

```diff
@@ -0,0 +1,182 @@
+import aloscene
+from aloscene import Depth, CameraIntrinsic, Flow, Mask
+from typing import Union
+import numpy as np
+import torch
+import torch.nn.functional as F
+
+
+def load_scene_flow(path: str) -> torch.Tensor:
+    with open(path, "rb") as file:
+        data = np.load(file)
+        return torch.from_numpy(data)
+
+
+class SceneFlow(aloscene.tensors.SpatialAugmentedTensor):
+    """
+    Scene flow map
+
+    Parameters
+    ----------
+    x : str
+        load scene flow from a numpy file
+    """
+
+    @staticmethod
+    def __new__(cls, x, occlusion: Mask = None, *args, names=("C", "H", "W"), **kwargs):
+        if isinstance(x, str):
+            # load flow from path
+            x = load_scene_flow(x)
+            names = ("C", "H", "W")
+
+        tensor = super().__new__(cls, x, *args, names=names, **kwargs)
+        tensor.add_child("occlusion", occlusion, align_dim=["B", "T"], mergeable=True)
+        return tensor
+
+    def __init__(self, x, *args, **kwargs):
+        super().__init__(x)
+
+    @classmethod
+    def from_optical_flow(
+        cls,
+        optical_flow: Flow,
+        depth: Depth,
+        next_depth: Depth,
+        intrinsic: CameraIntrinsic,
+        mode: str = "bilinear",
```

Comment:
- replace `mode` parameter by `sampling` for more explicit name. 
- add a description in the docstring